src/dbnode/network/server/tchannelthrift/node/service.go

```diff
@@ -66,6 +69,9 @@ const (
 	maxSegmentArrayPooledLength = 32
 	// Any pooled error slices that grow beyond this capcity will be thrown away.
 	writeBatchPooledReqPoolMaxErrorsSliceSize = 4096
+
+	fetchTaggedSpanID = "rpc_fetch_tagged"
```

Comment:
We should do some standardisation of the names we use when creating spans. The ideal is to have a easy 1-1 mapping from name to the code, without requiring users to look at the code itself (to make it easy to read the trace output in jaeger). 

I'd suggest a convention like: `packageName.objectName.Method` or `packageName.Method` in case there isn't an object. where packageName is abbreviated enough to make it clear without being too verbose. So in this case, it'd be `tchannelthrift/node.Service.Query` 

What do you think?

Reply:
let me read a bit about opentracing/jaeger naming conventions.

Reply:
how about keeping these names, but including tags? for example, this one would still be called `rpc_fetch_tagged`, but we could have a series of tags like:
```
method:query
package:node
component:tchannelthrift
```